src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -5,20 +5,23 @@ namespace System.Runtime.InteropServices.RuntimeInformationTests
 {
     public class CheckPlatformTests
     {
-        [Fact, PlatformSpecific(PlatformID.Windows)]
-        public void CheckWindows()
+        [Fact, PlatformSpecific(PlatformID.FreeBSD)]
```

Comment:
This compiles?  Don't the build tools first need to be updated to add this enum value and associated traits logic?